cypress.json

```diff
@@ -1,7 +1,6 @@
 {
   "video": false,
   "fixturesFolder": false,
```

Comment:
I added this line to avoid cypress creating an empty support file.
Now that we want to have a support file, we need to remove it, to let cypress load the proper file.